widgets/gui/qt_left_window.py

```diff
@@ -63,7 +66,21 @@ def __init__(self, parent):
                                [self, "stripe_reduction_offset", -10, 10, float, 0.01, 0.58]]
 
         for i in self.parameter_list:
-            self.layout.addWidget(qt_textbox_and_slider.TextboxAndSlider(*i))
+            textbox_and_slider = qt_textbox_and_slider.TextboxAndSlider(*i)
+            self.layout.addWidget(textbox_and_slider)
             self.layout.addWidget(qt_line_break.LineBreak(Qt.AlignTop))
 
+            self.parameter_vals.append(textbox_and_slider)
+
         self.setLayout(self.layout)
+
+        # not in property for testing purposes
+        for i in range(0, 14):
+            print(self.parameter_vals[i].layout.itemAt(1).itemAt(0).widget.Value)
+
+
+    #@property
+    #def update_parameters(self):
```

Comment:
I imagine having a get_parameters property can work better.

Reply:
Yes, slot and signal naming is poor at the moment. I will think it through when I get this working

Reply:
This small PR is another thing I tried, gives no error, but prints nothing, even though there should be quite a few widgets there